src/components/pages/blog/posts.tsx

```diff
@@ -4,10 +4,16 @@ import styled from 'styled-components';
 import { BlogPostTeaser } from '../../../types';
 import { Teaser } from '../../content/teaser/teaser';
 import { LONG_DATE_FORMAT, useLocaleFormat } from '../../i18n-helpers';
+import { up } from '../../support/breakpoint';
 
 const BlogTeaserGrid = styled.div`
   display: grid;
-  gap: 24px;
+  gap: 48px;
+
+  ${up('md')} {
+    gap: 24px;
```

Comment:
```suggestion
    row-gap: 60px;
    column-gap: 24px;
```

The gaps on desktop don't have the same values in the design

Reply:
fixed it